XamlControlsGallery/ControlPages/ImplicitTransitionPage.xaml

```diff
@@ -13,7 +13,7 @@
         </Style>
     </Page.Resources>
     <StackPanel Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-        <local:ControlExample HeaderText="Automatically animate changes to Opacity" >
+        <local:ControlExample HeaderText="Automatically animate changes to Opacity" MinimumUniversalAPIContract="7">
```

Comment:
Why do we need this? Isn't the point of WinUI 3 that we are detached from the OS or is this one of the areas where this does not apply?